lib/pdf/reader/standard_security_handler_v5.rb

```diff
@@ -0,0 +1,89 @@
+# coding: utf-8
+require 'digest'
+require 'openssl'
+
+class PDF::Reader
+
+  # class creates interface to encrypt dictionary for use in Decrypt
+  class StandardSecurityHandlerV5
+
+    attr_reader :key_length, :encrypt_key
+
+    def initialize(opts = {})
+      @key_length   = 256
+      @O            = opts[:O]   # hash(32B) + validation salt(8B) + key salt(8B)
+      @U            = opts[:U]   # hash(32B) + validation salt(8B) + key salt(8B)
+      @OE           = opts[:OE]  # decryption key, encrypted w/ owner password
+      @UE           = opts[:UE]  # decryption key, encrypted w/ user password
+      @encrypt_key  = build_standard_key(opts[:password] || '')
+    end
```

Comment:
I notice that `:EncryptMetadata` is not passed in here - is the `:AESV3` algorithm the same in both cases?